Positioning2/style.css

```
@@ -0,0 +1,66 @@
+.parent {
+    background-color: rgb(230, 10, 10)7;
+    display: flex;
+    width: 555px;
+    height: 191px;
+
+    border-style: dotted;
+    border-color: rgb(87, 67, 93);
+    border-radius: 10px;
+    border-width: 5px;
+
+    top: 75px;
+
+    text-align: center;
+}
+
+.parent {
+    background-color: rgb(168, 176, 167);
+    top: 0;
+    left: 50%;
+}
+
+h2 {
+    display: flex;
+    justify-content: center;
+    padding-top: 20px;
+}
```

Pentru a centra textul e de ajuns text-aling;

```suggestion
h2 {
    text-align: center;
}
```